### ISSUE TITLE
Bug report: Cannot run gpg, Cask doesn't know it's tapped

### ISSUE THREAD

**Issue by chrisfinazzo** [caskroom/homebrew-cask#24134]
#### Description of issue

As described in #24072, whenever I submit a Cask to be updated through the automated mechanism, it fails with a message about the remotes not existing. 

<details><summary>Output of `brew doctor`</summary>

```
Chris-Finazzos-MacBook-Pro:~ Chris$ brew doctor
Your system is ready to brew.
```
</details>

<details><summary>Output of `brew cask doctor`</summary>

```
Chris-Finazzos-MacBook-Pro:~ Chris$ brew cask doctor
==> macOS Release:
10.11.6
==> Hardware Architecture:
intel-64
==> Ruby Version:
2.0.0-p648
==> Ruby Path:
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
==> Homebrew Version:
Homebrew 0.9.9 (git revision 8e29cf; last commit 2016-08-28)
Homebrew/homebrew-core (git revision 7b73; last commit 2016-08-29)
==> Homebrew Executable Path:
/usr/local/bin/brew
==> Homebrew Cellar Path:
/usr/local/Cellar
==> Homebrew Repository Path:
/usr/local
==> Homebrew Origin:
https://github.com/Homebrew/brew.git
==> Homebrew-Cask Version:
0.60.0 (git revision 51035; last commit 5 hours ago)
==> Homebrew-Cask Install Location:
<NONE>
==> Homebrew-Cask Staging Location:
/usr/local/Caskroom
==> Homebrew-Cask Cached Downloads:
/Users/Chris/Library/Caches/Homebrew/Cask
0 files, 0B
==> Homebrew-Cask Default Tap Path:
/usr/local/Library/Taps/caskroom/homebrew-cask
==> Homebrew-Cask Alternate Cask Taps:
==> Homebrew-Cask Default Tap Cask Count:
3277
==> Contents of $LOAD_PATH:
/usr/local/Library/Homebrew/cask/lib
/usr/local/Library/Homebrew
/Library/Ruby/Site/2.0.0
/Library/Ruby/Site/2.0.0/x86_64-darwin15
/Library/Ruby/Site/2.0.0/universal-darwin15
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0/x86_64-darwin15
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0/universal-darwin15
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/x86_64-darwin15
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/universal-darwin15
==> Contents of $RUBYLIB Environment Variable:
<NONE>
==> Contents of $RUBYOPT Environment Variable:
<NONE>
==> Contents of $RUBYPATH Environment Variable:
<NONE>
==> Contents of $RBENV_VERSION Environment Variable:
<NONE>
==> Contents of $CHRUBY_VERSION Environment Variable:
<NONE>
==> Contents of $GEM_HOME Environment Variable:
<NONE>
==> Contents of $GEM_PATH Environment Variable:
<NONE>
==> Contents of $BUNDLE_PATH Environment Variable:
<NONE>
==> Contents of $PATH Environment Variable:
PATH="/Users/Chris/.rbenv/shims:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin:/usr/local/Library/Homebrew/cask/cmd:/usr/local/Library/Homebrew/shims/scm"
==> Contents of $SHELL Environment Variable:
SHELL="/bin/bash"
==> Contents of Locale Environment Variables:
LANG="en_US.UTF-8"
==> Running As Privileged User:
No
```
</details>

The line `==> Homebrew-Cask Install Location:
<NONE>` makes me think that this is an artifact of the Caskroom migration as `brew tap` shows the following:

<details><summary>Output of `brew tap`</summary>

```
Chris-Finazzos-MacBook-Pro:~ Chris$ brew tap
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
No changes to formulae.
caskroom/cask
homebrew/completions
homebrew/core
homebrew/dupes
vitorgalvao/tiny-scripts
```
</details>

As a result, when I attempt to submit a Cask the following errors occur: 

 <img width="697" alt="screen shot 2016-08-29 at 11 42 38 am" src="https://cloud.githubusercontent.com/assets/1534882/18058310/c34c4e4c-6de1-11e6-9dea-bf0ca3404a04.png">

<img width="697" alt="screen shot 2016-08-29 at 11 43 19 am" src="https://cloud.githubusercontent.com/assets/1534882/18058323/cc6361f0-6de1-11e6-859a-1168cb59b4be.png">

**Comment by adidalal** [comment]
Thanks for the detailed bug report.
It looks like HBC might not have been forked and the remote url set properly.

Can you:
Delete and refork the repo from github (just to kill any errant branches)
`cd "$(brew --repository)"/Library/Taps/caskroom/homebrew-cask/Casks; git checkout master`
and check the output of `git remote -v`


Should be something like:

```
adidalal	https://github.com/adidalal/homebrew-cask.git (fetch)
adidalal	https://github.com/adidalal/homebrew-cask.git (push)
origin	https://github.com/caskroom/homebrew-cask (fetch)
origin	https://github.com/caskroom/homebrew-cask (push)
```

If not, do `git remote add chrisfinazzo https://github.com/chrisfinazzo /homebrew-cask.git`

That should fix things - let me know how it goes.

**Comment by chrisfinazzo** [comment]
Tl;dr I had an issue with `gpg2` that needed to be fixed. I may have accidentally set keys to expire previously without realizing. It looks like @mjsteinbaugh has the same PR elsewhere, pull as you see fit.
